packages/api/apollo-server/mutations/submission.ts

```diff
@@ -0,0 +1,35 @@
+import { submissions } from '../mock-data'
+
+export interface IEditSubmissions {
+  assignmentID: String
+}
+
+// edit all submissions related to an assignment
+// e.g. change Submission.reviewBy
+export const editSubmissions = async (_: any, args: IEditSubmissions) => {
+  console.log(args)
+  return submissions
+}
+
+export interface IAddAnnotation {
+  fileID: String
+  givenByID: String
+  line: Number
+  content: String
+}
+
+export const addAnnotation = async (_: any, args: IAddAnnotation) => {
+  return args
+}
+
+export interface IEditAnnotation {
+  id: String
+  givenById?: String
+  line?: Number
+  content?: String
```

Comment:
we should only need to edit the content. In any other case, we would replace the annotation with a new one.